model/service.go

```diff
@@ -67,10 +92,13 @@ type ServiceInstance struct {
 
 // Port represents a network port
 type Port struct {
-	// Port value (may be distinct from the service port with the same name)
-	Port int `json:"port"`
-	// Name of the port classifies ports for a single service, optional
+	// Name of the port classifies ports for a single service, optional only for
+	// a single port service
 	Name string `json:"name,omitempty"`
+
```

Comment:
per offline discussion, this is once again a K8S specific abstraction, thats leaking into the generic Istio model of a service. I don't have a good solution for this at this hour. Pushing it down into platform/kube.go may make the code there ugly, but keeps interfaces clean. Keeping name field here is fine, as long as its a benign entry in the port spec, but there needs to be an API to retrieve ports by Protocol or Port or Name.